### ISSUE TITLE
Footer icons looks ruined after upgrading to Kubuntu 21.10

### ISSUE THREAD

**Issue by aliquo93** [prateekmedia/Menu11#47]
As the title says, the footer icons look ruined after upgrading to Kubuntu 21.10 (with 21.04 they used to look fine).

What can I do? I've already tried to reinstall the applet but with no success. Maybe there is some configuration file I have to delete to completely reset the applet to its default settings?

Additional infos:

- Kernel: 5.13.0-27-generic
- Plasma version: 5.23.5
- KDE Frameworks version: 5.90.0
- Qt Version: 5.15.2

I also attach a screenshot:

![footer](https://user-images.githubusercontent.com/9071297/150917480-1f46106c-2e5a-47b2-8634-17cce4eca236.png)

**Comment by prateekmedia** [comment]
That's theme issue to be exact. Try changing themes?

**Comment by aliquo93** [comment]
> That's theme issue to be exact. Try changing themes?

Nope, it's the same exact theme if was using on 21.04.

**Comment by aliquo93** [comment]
I've update the first post with additional infos.

**Comment by prateekmedia** [comment]
Sorry misclicked close issue button, Can you try if it is same with Brreze theme?

**Comment by aliquo93** [comment]
> 

Yes, it's less noticeable, but if you pay enough attention you can still see those glitchy squares around the icons.

![footer2](https://user-images.githubusercontent.com/9071297/150919479-745beb23-5fb7-40b2-973a-e43e60423b76.png)

**Comment by aliquo93** [comment]
No solution to this?

**Comment by prateekmedia** [comment]
Hi @aliquo93, Actually I am not on KDE for now, till then I am pinning the issue so someone can help you out or create a PR until I come back.

**Comment by aliquo93** [comment]
Thank you!
